doc/source/conf.py

```diff
@@ -145,26 +201,26 @@
 
 # If true, SmartyPants will be used to convert quotes and dashes to
 # typographically correct entities.
-#html_use_smartypants = True
+# html_use_smartypants = True
 
 # Custom sidebar templates, maps document names to template names.
 #html_sidebars = {}
 html_sidebars = {
-   '**': ['localtoc.html', "relations.html", 'quicklinks.html', 'searchbox.html', 'editdocument.html'],
```

Comment:
`doc/source/_templates/quicklinks.html` still exists, it can be deleted.

Reply:
I had left out that one because I wasn't sure if the newer quick links were looking okay – addressed in 1b18125f692385af11f490344a062673c8a523d5